app/models/katello/marketing_engineering_product.rb

```diff
@@ -1,9 +0,0 @@
-module Katello
-  class MarketingEngineeringProduct < Katello::Model
```

Comment:
Sorry, to be clear, I mean do you need to include a database migration to drop this table? (I didn't see one)